unit/c2goto/library/builtin_libs.fuzz.cpp

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************
+ Module: Builtin Libs Model unit test
+
+ Author: Rafael Sá Menezes
+
+ Date: December 2019
+
+ Fuzz Plan:
+   - Functions
+ \*******************************************************************/
+
+#include <stdlib.h>
+#include <cassert>
+
+
+void __ESBMC_atomic_begin() {}
+void __ESBMC_atomic_end() {}
+extern int __builtin_esbmc_sync_fetch_and_add(int*, int);
```

Comment:
you probably don't need these declarations

Reply:
This is a fuzz test for a model. I think that this is needed since we need to provide these declarations some way